frameworks.rst

```diff
@@ -67,7 +64,7 @@ links!
 `QWeb <https://github.com/antonylesuisse/qweb>`_
     Another WSGI framework (not sure what the distinguishing features
     are)
-`repoze.zope2 <http://repoze.org>`_ 
+`repoze.zope2 <https://pypi.org/project/repoze.zope2/>`_ 
```

Comment:
Should probably be moved to deprecated (or removed), according to [the repoze documentation](https://repoze.readthedocs.io/en/latest/components.html) it's

> Obsoleted by [Zope 2.13.x release](https://pypi.python.org/pypi/Zope2)

Also the lists are currently correctly ordered, so this should probably be maintained :)

Reply:
I think it's best to remove the repoze.zope2 part.
I will be waiting for this change (removal of the entry) in this commit for you approval.

Yes, changing the order of the lists is not a good idea.
thanks for the review and the suggestions! I'm waiting for your response. Thank you so much again!